assets/app/styles/_core.less

```diff
@@ -483,8 +483,13 @@ label.checkbox {
   margin-left: 3px;
 }
 
-// Misc
+
+// Misc 
+// TODO: pull out into _partials where it makes sense 
 // --------------------------------------------------
+a[ng-click] {
+  cursor: pointer;
+}
```

Comment:
I believe the link can't be activated from the keyboard as well without an `href`.